### DISCUSSION TITLE
Vite 5 is out!

### DISCUSSION THREAD

**Discussion by knjshimi** [barrel/shopify-vite#110] [General]
Hey Barrel team, do you think it's safe to use vite 5 with vite-plugin-shopify?

Maybe I should have tested it myself, but I have kept myself busy trying to migrate heaps of projects from webpack/theme-kit to vite/vite-plugin-shopify/shopify-cli.

Your work is deeply appreciated! Thanks heaps!

**Comment by montalvomiguelo** [comment]
Hi @kenji-twp (: 
Not yet; the path to read the manifest.json file needs to be adjusted per the updates in Vite 5
https://github.com/vitejs/vite/commit/74fa024db4aaf04d0bdc6b6ec87e3095e1b24b12

**Reply by knjshimi** [reply]
Hey @montalvomiguelo, I saw that you have merged the PR that adds vite 5 support, but it seems there's no new release? Maybe I'm missing something... :'(

**Reply by montalvomiguelo** [reply]
the release v3 has just been published (:

**Reply by knjshimi** [reply]
Thanks @montalvomiguelo! Would I be pedantic if I asked for an update on the page reload plugin? 🥺

**Reply by montalvomiguelo** [reply]
The page reload plugin has been updated (: 
https://github.com/barrel/shopify-vite/releases/tag/vite-plugin-page-reload%400.2.0
